styles.css

```diff
@@ -9,17 +9,75 @@ html {
   font-size: calc(15px + 0.390625vw);
 }
 
+body {
+  padding: 10px;
+}
+
+a {
+  text-decoration: none;
+}
+
+ul {
+  list-style-type: none;
+}
+
+.page {
+  display: none;
+  min-height: calc(100vh - 20px);
+  position: relative;
+}
+
+.page.active {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.top-nav {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0.5rem 0.7rem;
+  border: 1px solid #4d6d80;
+}
+
+.top-nav h3 {
+  font-size: 0.85rem;
+}
+
+.top-nav ul {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.top-nav ul li {
+  margin-left: 10px;
+}
+
+.top-nav ul li:not(:last-child) {
+  border-right: 0.07rem solid #4d6d80;
+  padding-right: 0.5rem;
+}
+
+.top-nav ul li a {
+  color: #000;
+  font-size: 0.8rem;
+}
```

Comment:
- [ ] [OPTIONAL] Good job so far! Please add `hover` and `active` effect to the navigation items as shown in the wireframe.

![image](https://user-images.githubusercontent.com/63915024/182649468-f520e9da-27f3-4824-ad5d-5aba0182c5d1.png)

- ### Your design:

![image](https://user-images.githubusercontent.com/63915024/182655368-17f100eb-04a6-48b2-bdff-db1c9b10dab1.png)